docs/content/server/getting-started.md

```diff
@@ -1,9 +1,15 @@
-# Setting up the server
+# Getting Started with AutoGPT: Self-Hosting Guide
+
+This tutorial will walk you through the process of setting up AutoGPT locally on your machine.
+
+<center><iframe width="560" height="315" src="https://www.youtube.com/embed/4Bycr6_YAMI?si=dXGhFeWrCK2UkKgj" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe></center>
+
+## Setting up the server
```

Comment:
This is the only H2 header now. If you remove this, all the headers below can stay the same level/size.